pkg/env/mongo.go

```diff
@@ -0,0 +1,49 @@
+package env
```

Comment:
Michal added package "config" in his aws s3 storage PR. We should agree on a common name...